src/pages/dataTableUtilities/getColumns.js

```diff
@@ -176,9 +198,17 @@ const PAGE_COLUMNS = {
     COLUMN_NAMES.STATUS,
     COLUMN_NAMES.REMOVE,
   ],
+  [ROUTES.STOCKTAKES_WITH_PROGRAMS]: [
+    COLUMN_NAMES.NAME,
+    COLUMN_NAMES.PROGRAM,
+    COLUMN_NAMES.CREATED_DATE,
+    COLUMN_NAMES.STATUS,
+    COLUMN_NAMES.REMOVE,
+  ],
   [ROUTES.STOCKTAKE_EDITOR]: [
     COLUMN_NAMES.ITEM_CODE,
     COLUMN_NAMES.ITEM_NAME,
+    COLUMN_NAMES.UNIT,
```

Comment:
offroad?

Reply:
Just reordering of columns, I think the diff here is a bit misleading!